Modules/blake2module.c

```diff
@@ -936,13 +983,15 @@ static PyType_Slot blake2s_type_slots[] = {
 static PyType_Spec blake2b_type_spec = {
     .name = "_blake2.blake2b",
     .basicsize =  sizeof(Blake2Object),
-    .flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_IMMUTABLETYPE,
+    .flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_IMMUTABLETYPE
+             | Py_TPFLAGS_HAVE_GC | Py_TPFLAGS_HEAPTYPE,
```

Comment:
`Py_TPFLAGS_HEAPTYPE` isn't needed

Reply:
Personally I think it's clearer that way (or is the value already set by another flag?)

Reply:
it is set by typeobject during type ready

Reply:
`PyType_FromMetaclass` (= `PyType_FromModuleAndSpec`) is a function to create a heaptype, which sets `Py_TPFLAGS_HEAPTYPE` before the `PyType_Ready()` call.  It looks like this PR is about converting the existing non-GC heap types to regular GC heap types.

Reply:
> is a function to create a heaptype, which sets

Ah I didn't know! I always thought that it *assumed* the flag to be set (I wouldn't expect the flags to actually change in general). I still think it's better to be explicit in general, both for the reader and for possible future changes but I'm also ok with removing this.